backend/middleware/exception.py

```diff
@@ -73,27 +74,30 @@ def process_exception(
 
     @staticmethod
     def format_exc_and_log(
-        request: HttpRequest, response: HttpResponse, exception: Exception
+        request: Request, response: Optional[Response], exception: Exception
```

Comment:
@chandrasekharan-zipstack why the type here was changed from HttpRequest to Request

Reply:
@ritwik-g checked during runtime, the types are that of DRF (`rest_framework`)

Reply:
Okay